docs/standard/class-libraries.md

```diff
@@ -42,7 +42,7 @@ Many library developers have switched from producing multiple platform-specific
 
 .NET Core libraries are a replacement of the platform-specific and portable libraries concepts. They are platform-specific in the sense that they expose all functionality from the underlying platform (no synthetic platforms or platform intersections). They are portable in the sense that they work on all supporting platforms.
 
-.NET Core exposes a set of library _contracts_. .NET platforms must support each contract fully or not at all. Each platform, therefore, supports a set of .NET Core contracts. The corollary is that each .NET Core class library is supported on the platforms that support it’s contract dependencies.
+.NET Core exposes a set of library _contracts_. .NET implementations must support each contract fully or not at all. Each implementation, therefore, supports a set of .NET Core contracts. The corollary is that each .NET Core class library is supported on the platforms that support it’s contract dependencies.
```

Comment:
.NET Core exposes --> The .NET Standard 
.NET Core contracts --> .NET Standard contracts
.NET Core class library --> .NET Standard class library